content/engineering/our-approach/release-strategies.md

```diff
@@ -10,7 +10,7 @@ layout: layouts/page
 eleventyNavigation:
   parent: engineering_approach
   key: Releasing software
-  order: 10
+  order: 9
```

Comment:
three different pages currently have `order: 10`, which isn't helpful for setting page order!